storage_image_product/models/product_product.py

```diff
@@ -11,15 +11,20 @@ class ProductProduct(models.Model):
     # small and medium image are here to replace
     # native image field on form and kanban
     variant_image_small_url = fields.Char(
-        related="variant_image_ids.image_id.image_small_url", store=True
+        related="variant_image_ids.image_id.image_small_url",
+        store=True,
+        string="Variant Image Small Url",
     )
     variant_image_medium_url = fields.Char(
-        related="variant_image_ids.image_id.image_medium_url", store=True
+        related="variant_image_ids.image_id.image_medium_url",
+        store=True,
+        string="Variant Image Medium Url",
```

Comment:
The string is required? I thought it was not needed when it was the same as the field name?

Reply:
@florian-dacosta The problem is the related as it takes the string from that related field ... which is the same as the existing 'image_medium_url' field